rector.php

```diff
@@ -17,16 +17,43 @@
 
 use Rector\Config\RectorConfig;
 use Rector\Php74\Rector\LNumber\AddLiteralSeparatorToNumberRector;
+use Rector\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector;
 use Rector\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector;
+use Rector\ValueObject\PhpVersion;
+use Ssch\TYPO3Rector\CodeQuality\General\ConvertImplicitVariablesToExplicitGlobalsRector;
+use Ssch\TYPO3Rector\CodeQuality\General\ExtEmConfRector;
+use Ssch\TYPO3Rector\Configuration\Typo3Option;
+use Ssch\TYPO3Rector\Set\Typo3LevelSetList;
+use Ssch\TYPO3Rector\Set\Typo3SetList;
 
 return RectorConfig::configure()
     ->withPaths([
         __DIR__ . '/.',
     ])
+    ->withImportNames()
     ->withPhpSets()
     ->withAutoloadPaths([
         __DIR__ . '/.Build/vendor/autoload.php',
     ])
+    ->withPhpVersion(PhpVersion::PHP_82)
+    ->withSets([
+        Typo3SetList::CODE_QUALITY,
+        Typo3SetList::GENERAL,
+        Typo3LevelSetList::UP_TO_TYPO3_13,
+    ])
+    // To have a better analysis from PHPStan, we teach it here some more things
+    ->withPHPStanConfigs([
+        Typo3Option::PHPSTAN_FOR_RECTOR_PATH,
+    ])
+    ->withRules([
+        AddVoidReturnTypeWhereNoReturnRector::class,
+        ConvertImplicitVariablesToExplicitGlobalsRector::class,
+    ])
+    ->withConfiguredRule(ExtEmConfRector::class, [
+        ExtEmConfRector::PHP_VERSION_CONSTRAINT => '8.2.0-8.3.99',
```

Comment:
This seems not to have any effect? At least php constraint is not available in ext_emconf.php. When adding this to ext_emconf, I suggest to use 8.9.99 instead of 8.3.99 to be in line with composer.json (^8.2).

Reply:
It seems like it only has an effect when there already is a php constraint. Added it now. Though as it is the same as the TYPO3 version not sure it is needed?

Reply:
I am fine removing it, then also the rector rule for that.

Reply:
Removed it